docs/api/migration-guides/local-simulators.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Migrate to local simulators
+description: Migrate from using the IBM Quantum cloud simulators (such as ibmq_qasm_simulator) to using local simulators.
```

Comment:
```suggestion
description: Migrate from using the IBM Quantum cloud simulators (such as `ibmq_qasm_simulator`) to using local simulators.
```

I assume we can use code syntax in metadata descriptions but lmk if im wrong 🤔

Reply:
It seems unnecessary to add code syntax in text that's not actually meant to be seen, so I just changed the text.